lib/rspec/matchers/built_in/be.rb

```diff
@@ -265,6 +265,8 @@ def validity_message
 
           msg = "expected #{@actual} to respond to `#{predicate}`"
           msg << " but `#{predicate}` is a private method" if private_predicate?
+          msg << " or perhaps you meant `be true` or `be_truthy`" if predicate == :true?
+          msg << " or perhaps you meant `be false` or `be_falsey`" if predicate == :false?
```

Comment:
If `true?` or `false` is a private predicate would we want both the `but #true? is a private method` and the `or perhaps you meant "be true" or "be truthy"` parts?  I think thinking that we wouldn't.  Maybe these 3 conditionals should be made exclusive, like so?

``` ruby
if private_predicate?
  msg << ...
elsif predicate == :true?
  msg << ....
elsif predicate == :false?
  msg << ...
end
```